### ISSUE TITLE
Can you please make BGM/no BGM builds with different titleIDs?

### ISSUE THREAD

**Issue by ksanislo** [RedInquisitive/Super-Haxagon#5]
Currently you have a shared title ID on both of these builds, they really should be different since they're different releases. It's hard to keep things properly indexed in the [3DS Homebrew Title Database](https://titledb.com) when they're not unique.

Also, currently your SMDH Name is set as "Super" and the description is set as "A" so you probably need to add some quotes around those names in the the Makefile.

If you make a new github release, the TitleDB will automatically pick up the corrections within about 5 minutes.

**Comment by RedTopper** [comment]
I mean, they are the same game, so why would I want them to have different title IDs? I don't think anyone should have both of them installed at the same time on the same 3ds. It's available so that you don't have to download the music if you don't want it. Tbh I should probably move the music to the SD card so people can change it but I don't know where to place it without cluttering up yet another folder.

Yeah, I'll check out the quote thing.

**Comment by ksanislo** [comment]
I have to enforce unique title id in the database so someone doesn't reuse
an ID for an alternate project. If you keep them the same I can only list
one for direct install.

**Comment by RedTopper** [comment]
Gotcha. I should really make only one build then. Let me tinker with make because it's giving me a headache right now, brb.

EDIT: The makefile won't compile with quotes... I'll try tic marks instead.

**Comment by RedTopper** [comment]
did that work?

**Comment by ksanislo** [comment]
I think I've botched the HTTP headers in passing to github while trying to
fix another .cia elsewhere that won't return the file if it doesn't think
I'm a web browser. So yours is not auto-updating yet, give me a few minutes
to figure out what I did wrong.

**Comment by ksanislo** [comment]
Alright, the first .cia is updating... I've got to rework the github scan, since previously I've only had to deal with multiple .cia files in an archive so I'd made a shortcut by not scanning further after reaching a .cia. I should have titledb supporting that second .cia within the next hour or so.

It may be a good idea to change the Description field on that second one to indicate it is without background music though, since the SMDH data is the only information that will be displayed to anyone using either TitleDB.com itself or any upcoming installers that utilize the API (e.g. Flask).
